domain/dcsa_ovs_domain.yaml

```diff
@@ -0,0 +1,294 @@
+            otherFacility:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/otherFacility'
+      required:
+        - transportCallID
+        - scheduleID
+        - carrierVoyageNumber
+        - UNLocationCode
+        - facilityTypeCode
+        - facilityCode
+    
+    ######################################
+    # Transport Event Related Objects
+    ######################################
+
+    transportEvent:
+      description:  A list of all the transport-events involved in a portCall.
+      type: object
+      allOf:
+        - $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_NOTIFICATION_DOMAIN/1.0.0#/components/schemas/transportNotification'
+        - type: object
+          properties:
+            locationType:
+              $ref: '#/components/schemas/locationType'
+        - type: object
+          properties:
+            locationID:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/locationID'
+          
+          
+#        - type: object
+#          properties:
+#            eventID:
+#              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/eventID'
+#        - type: object
+#          properties:
+#            eventType:
+#              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/eventType'
+#        - type: object
+#          properties:
+#            eventDateTime:
+#              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/eventDateTime'
+#        - type: object
+#          properties:
+#            eventClassiefierCode:
+#              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_NOTIFICATION_DOMAIN/1.0.0#/components/schemas/eventClassifierCode'
+#        - type: object
+#          properties:
+#            EventTypeCode:
+#              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/eventTypeCode'
+#        - type: object
+#          properties:
+#            transportCallID:  
+#              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/transportCallID'
+#        - type: object
+#          properties:
+#            locationType:
+#              $ref: '#/components/schemas/locationType'
+#        - type: object
+#          properties:
+#            locationID:
+#              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/locationID'
+#        - type: object
+#          properties:
+#            comment:
+#              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/eventComment'
+#        - type: object
+#          properties:
+#            delayReasonCode:
+#              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/delayReasonCode'
```

Comment:
```suggestion
```

Reply:
Any need for this?

Reply:
I kept it in for discussion concerning the event / notfication topic, in the current implemenation I'm using events but in this schema I already use notifications from the Notification Domaain. 
Removed it, as we have the discussion ongoing